rts/motoko-rts/src/gc/mark_compact/bitmap.rs

```diff
@@ -157,6 +158,21 @@ pub unsafe fn iter_bits() -> BitmapIter {
 pub const BITMAP_ITER_END: u32 = u32::MAX;
 
 impl BitmapIter {
+    pub unsafe fn advance(&mut self, address: u32) {
```

Comment:
Thank you very much. Very good point. I removed the bitmap as it is actually no longer really since I reduced the bitmap size for the young generation collection (as suggested by Claudio).